crates/query-engine/translation/src/translation/mutation/v1/insert.rs

```diff
@@ -78,21 +78,34 @@ pub fn translate(
 
     check_columns(&mutation.columns, &columns, &mutation.collection_name)?;
 
-    let insert = ast::Insert {
+    // We add an always true constraint check to unify the mutations interface.
+    let check_constraint_alias =
+        sql::helpers::make_column_alias(sql::helpers::CHECK_CONSTRAINT_FIELD.to_string());
+    let check_constraint_value = sql::helpers::true_expr();
+
+    let insert = sql::ast::Insert {
         schema: mutation.schema_name.clone(),
         table: mutation.table_name.clone(),
         columns,
         values,
-        returning: ast::Returning::ReturningStar,
+        // RETURNING *, true
+        returning: sql::ast::Returning::Returning(sql::ast::SelectList::SelectListComposite(
+            Box::new(sql::ast::SelectList::SelectStar),
+            Box::new(sql::ast::SelectList::SelectList(vec![(
+                check_constraint_alias.clone(),
+                check_constraint_value,
+            )])),
+        )),
     };
-    Ok(insert)
+
+    Ok((insert, check_constraint_alias))
 }
 
 /// Check that no columns are missing, and that columns cannot be inserted to
 /// are not insertred.
 fn check_columns(
     columns: &BTreeMap<String, database::ColumnInfo>,
-    inserted_columns: &[ast::ColumnName],
+    inserted_columns: &[sql::ast::ColumnName],
```

Comment:
Would it be possible to do this import renaming pass in a PR before this one? There is a lot going on in this PR without (absolutely sensible, granted) changes like these.

Reply:
OK